### ISSUE TITLE
V-Google. Looks like the domain name has expired.

### ISSUE THREAD

**Issue by Jesse205** [vuejs/awesome-vue#4136]
![image](https://github.com/vuejs/awesome-vue/assets/51242302/6318e991-14ac-41a3-91c6-2afdbf6bc568)
![image](https://github.com/vuejs/awesome-vue/assets/51242302/8855cc91-69bd-4d66-87b1-b56008ce39fb)

它现在是一个赌博网站，且网站似乎也不是Vue做的。

It is now a gambling site and the site does not appear to be made by Vue.

**Comment by loicduong** [comment]
Are you talking about [v-oogle](https://github.com/VeryWow/v-oogle)? If so, your domain is not the correct one for this project. It should actually be https://verywow.github.io/v-oogle/

**Comment by Jesse205** [comment]
@loicduong Yes, I was referring to v-oogle. but the readme for this repository points v-oogle to a gambling site.

**Comment by loicduong** [comment]
I just checked and there are 2 v-oogle in the README.md. One of them navigates just like you said.

**Comment by Jesse205** [comment]
@loicduong I think these two should be the same project, it just appears twice in the readme because the domain v-google.com was removed in the latest commit from https://github.com/VeryWow/v-oogle.
